### PR TITLE
fix: security issues caused by outdated dependencies

### PR DESCRIPTION
**Pull request by bufferoverflow**
gemnasium reported some [issues](https://gemnasium.com/omniauth/omniauth-saml/alerts)

### CODE REVIEW

File: omniauth-saml.gemspec [hunk 1]
```diff
@@ -12,11 +12,12 @@ Gem::Specification.new do |gem|
   gem.homepage      = 'https://github.com/omniauth/omniauth-saml'
 
   gem.add_runtime_dependency 'omniauth', '~> 1.3'
-  gem.add_runtime_dependency 'ruby-saml', '~> 1.1', '>= 1.1.1'
+  gem.add_runtime_dependency 'ruby-saml', '~> 1.1'
```
Comment (md5) [inline omniauth-saml.gemspec]: Can you revert this line? There were some issues with ruby-saml 1.1.0, so I consider this a regression.

**Comment by bufferoverflow** [comment]
updated the PR, coverage decreased because of newer coverage gem.

**Comment by md5** [comment]
@bufferoverflow This looks OK to me. One thing I'm not sure about is whether `coveralls` should be removed from the `Gemfile`. I put it over there so that it could be in the `test` group instead of `development`, but I'm not sure it matters.

**Comment by bufferoverflow** [comment]
The gem specification has only [add_development_dependency](http://guides.rubygems.org/specification-reference/#add_development_dependency) so I guess we should just merge this.

**Comment by supernova32** [comment]
Thanks @bufferoverflow for updating the dependencies. I agree we can merge this.

**Comment by md5** [comment]
@bufferoverflow Sounds good. That being said, I think that having `coveralls` in the `Gemfile` as well as the `*.gemspec` is now redundant.

**Pull request merged**